### ISSUE TITLE
Remove usage of `componentWillMount` for React 17

### ISSUE THREAD

**Issue by gbedardsiceupgrade** [davidtheclark/react-aria-menubutton#101]
`componentWillMount` triggers warnings in `react@16.9.x`, and won't be supported in `react@17.x.x`.

```
Warning: componentWillMount has been renamed, and is not recommended for use. See https://fb.me/react-async-component-lifecycle-hooks for details.

* Move code with side effects to componentDidMount, and set initial state in the constructor.
* Rename componentWillMount to UNSAFE_componentWillMount to suppress this warning in non-strict mode. In React 17.x, only the UNSAFE_ name will work. To rename all deprecated lifecycles to their new names, you can run `npx react-codemod rename-unsafe-lifecycles` in your project source folder.

Please update the following components: AriaMenuButtonButton, AriaMenuButtonWrapper
```

**Comment by shirshendubhowmick** [comment]
Aware of this, will be fixed soon

**Comment by shirshendubhowmick** [comment]
Fixed in #94
